README.md

```diff
@@ -192,95 +192,102 @@ In many cases a developer may find that the package's out-of-the-box `bgp.DEFAUL
+        "bytes": "r(xml bytes)"
       }
-    }
-  },
-  "fulltext": {
-    "total_time": "r(fulltext processor process seconds)",
-    "modules": {
-      "readinglevel": {
-        "time": "r(reading level process seconds)",
-        "results": {
-          "readability": {
-            "flesch_kincaid_score": "r(flesch kincaid score)",
-            "smog_score": "r(smog score)"
+    },
+    "processors": {
+      "1gram": {
+        "tokenization_time": "r(1gram tokenization process seconds)",
+        "total_tokens": "r(1gram count)",
+        "total_time": "r(1gram processor process seconds)",
+        "modules": {
+          "urls": {
+            "time": "r(url process seconds)"
           },
-          "lexile": {
-            "min_age": "(Lower age in range)",
-            "max_age": "(Upper age in range)"
+          "1grams": {
+            "time": "r(1gram frequency process seconds)"
+          }
+        }
+      },
+      "2gram": {
+        "tokenization_time": "r(2gram tokenization process seconds)",
+        "total_tokens": "r(2gram count)",
+        "total_time": "r(2gram processor process seconds)",
+        "modules": {
+          "2grams": {
+            "time": "r(2gram frequency process seconds)"
+          }
+        }
+      },
+      "fulltext": {
+        "total_time": "r(fulltext processor process seconds)",
+        "modules": {
+          "readinglevel": {
+            "time": "r(reading level process seconds)"
+          }
+        }
+      },
+      "pagetypes": {
+        "total_time": "r(pagetype processor process seconds)",
+        "modules": {
+          "copyright_page": {
+            "time": "r(copyright page process seconds)"
+          },
+          "backpage_isbn": {
+            "time": "r(copyright page process seconds)"
           }
         }
       }
     }
   },
-  "pagetypes": {
-    "total_time": "r(pagetype processor process seconds)",
-    "modules": {
-      "copyright_page": {
-        "time": "r(copyright page process seconds)",
-        "results": [
-          {
-            "page": "(copyright page)",
-            "isbns": [
+  "urls": [
+              "(url)"
+            ],
+  "1grams": [
+              [
+                "(1gram)",
+                "r(1gram frequency)"
+              ]
+            ],
+  "2grams": [
+              [
+                "(2gram)",
+                "r(2gram frequency)"
+              ]
+            ],
+  "readinglevel": {
+              "readability": {
+                "flesch_kincaid_score": "r(flesch kincaid score)",
+                "smog_score": "r(smog score)"
+              },
+              "lexile": {
+                "min_age": "(Lower age in range)",
+                "max_age": "(Upper age in range)"
+              }
+            },
+  "copyright_page": [
+              {
+                "page": "(copyright page)",
+                "isbns": [
+                  "(isbn)"
+                ]
+              }
+            ],
```

Comment:
Not totally confident in this schema. I was imagining having a `copyright_isbn` field with just the isbn and no page number but maybe that doesn't make sense. This might be good enough.

Reply:
copyright_page seems like it should probably be a page_type

Reply:
The page type is indicated by the module name "copyright_page". A different page type identifier such as Table of Contents would be contained in a different module named something like "toc_page". I see how the schema here is not clearly defined. I'll fix it.